flocker/ca/_ca.py

```diff
@@ -216,7 +312,7 @@ def initialize(cls, path, name):
             )
 
         dn = DistinguishedName(commonName=name)
-        keypair = generate_keypair()
```

Comment:
Yeah, I'd prefer `flocker_keypair()` and it should return an instance of ...let's call it `ComparableKeyPair` whose `keypair` attribute is a readonly `@property` which returns `self.keypair` `.original` ?? Not sure about that bit, or if there's an `IKeyPair` you could maybe use `proxyForInterface`.
Chat to @exarkun about what's be most  likely to be accepted back into Twisted.